RecoTracker/TkDetLayers/src/TkDetUtil.h

```diff
@@ -30,6 +44,219 @@ namespace tkDetUtil {
                            const TrajectoryStateOnSurface& ts,
                            const Plane& plane);
 
+  float computeYdirWindowSize(const GeomDet* det,
+                              const TrajectoryStateOnSurface& tsos,
+                              const MeasurementEstimator& est);
+
+  std::array<int, 3> findThreeClosest(std::vector<RingPar> ringParams,
+                                      std::vector<GlobalPoint> ringCrossing,
```

Comment:
```suggestion
  std::array<int, 3> findThreeClosest(const std::vector<RingPar>& ringParams,
                                      const std::vector<GlobalPoint>& ringCrossing,
```

Reply:
You're right, all the ones you mention should be fixed now